test/bench/helpers.hpp

```diff
@@ -38,7 +38,7 @@ inline void baseline_analyze(benchmark::State& state, bytes_view code) noexcept
     for (auto _ : state)
     {
         auto r = evmone::baseline::analyze(code.data(), code.size());
-        benchmark::DoNotOptimize(r);
+        benchmark::DoNotOptimize(&r);
```

Comment:
What does this change do?

Reply:
It does not like the `CodeAnalysis` now having `const` fields.

Reply:
If it's the same, maybe would make sense to chanbe advanced `analyze` above for consistency.